concurrent_test.go

```diff
@@ -511,6 +537,23 @@ func (rs historyRecords) Swap(i, j int) {
 	rs[i], rs[j] = rs[j], rs[i]
 }
 
+func validateIncrementalTxid(rs historyRecords, op OperationType) error {
```

Comment:
Operation type should be property of operation, not taken for granted for whole history
```suggestion
func validateIncrementalTxid(rs historyRecords) error {
```

Reply:
Yes, I thought about this. We can get the operationType from any record in `rs rs historyRecords`; The reason passing in a `op OperationType` is to explicitly say all records have the same operation type.

Reply:
removed the second parameter `op OperationType`